documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -1007,3 +1046,38 @@ remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
 
 Additionally, you might also want to enable client certificate authentication (and therefore also allow the use of the EXTERNAL SASL mech to authenticate clients).
 
+
+
+== Multi-tenancy
+
+Multi-tenancy allows accessing multiple containers as shown below:
+
+image::images/multi-tenancy.png.png[]
+
+Currently there are two supported protocols for accessing the data - using Hot Rod client and using REST interface.
```

Comment:
Can someone use a multi-tenant router and allow direct access to the HotRod/REST/Memcached servers at the same time?